src/Sylius/Component/Currency/spec/Converter/CurrencyConverterSpec.php

```diff
@@ -48,7 +48,15 @@ function it_throws_exception_if_currency_is_not_found($currencyRepository)
 
         $this
             ->shouldThrow(new UnavailableCurrencyException('EUR'))
-            ->duringConvert(6555, 'EUR')
+            ->duringConvertFromBase(6555, 'EUR')
         ;
     }
+
+    function it_converts_from_base_currency(CurrencyInterface $currency, $currencyRepository)
+    {
+        $currencyRepository->findOneBy(['code' => 'PLN'])->shouldBeCalled()->willReturn($currency);
+        $currency->getExchangeRate()->shouldBeCalled()->willReturn(0.24);
+
+        $this->convertToBase(6555, 'PLN')->shouldReturn(27313);
```

Comment:
Maybe you can figure out a simpler example, so that I could calculate that in the memory? :smile: 
```
$currency->getExchangeRate()->willReturn(0.25);
$this->convertToBase(100, 'PLN')->shouldReturn(4000);
```